chaos_kitten/brain/attack_planner.py

```
@@ -43,169 +62,95 @@ def __init__(self, endpoints: List[Dict[str, Any]], toys_path: str = "toys/") ->
-            self.load_attack_profiles()
-            
-        endpoint_path = endpoint.get("path", "")
-        method = endpoint.get("method", "").lower()
         
-        # Collect potential target fields from the endpoint definition
-        targetable_fields = [] # (name, location, type)
+        path = endpoint.get("path", "")
+        method = endpoint.get("method", "GET")
+        params = endpoint.get("parameters", [])
+        body = endpoint.get("requestBody", {})
+        cache_key = f"{method}:{path}:{str(params)}:{str(body)}"
```

Ruff (0.15.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)